introduction_to_amazon_algorithms/k_nearest_neighbors_covtype/k_nearest_neighbors_covtype.ipynb

```diff
@@ -2,7 +2,17 @@
  "cells": [
```

Comment:
<p>I commented out some of the extra permutations to shorten runtime. If you only want to run the one, please update the description to say <code>The following code will set up 1 endpoint</code>  instead of <code>The following code will set up 2 endpoints</code>  , and clarify the one machine and index used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Updated the description</p>